gcc/rust/lex/rust-lex.cc

```diff
@@ -2504,338 +2499,133 @@ Lexer::parse_char_or_lifetime (Location loc)
     }
 }
 
+// TODO remove this function
```

Comment:
Please open an issue so we don't forget :)

Reply:
Already raised in https://github.com/Rust-GCC/gccrs/issues/2309